### DISCUSSION TITLE
Starship not work in zsh with p10k installed

### DISCUSSION THREAD

**Discussion by Q-Bug4** [starship/starship#4533] [Q&A]
Starship not work after I add `eval "$(starship init zsh)"` into `~/.zshrc` while oh-my-zsh and p10k installed. So I backup my zshrc file and remove commands of p10k and oh-my-zsh. So here is my zshrc file now: 
```bash
plugins=(
    git
    z
)

export NO_PROXY=localhost,127.0.0.1
eval "$(starship init zsh)"
```

But after reset zsh, here it is.
![](https://i.imgur.com/L1wG4jH.png)

**Other Infomation**
```bash
uname -a
# Linux hasee 5.15.74-3-MANJARO #1 SMP PREEMPT Sat Oct 15 13:39:11 UTC 2022 x86_64 GNU/Linux

```

**Comment by davidkna** [comment]
`prompt off` will often work to disable other zsh prompts, but I don't know if that works for p10k in particular, but it should work with omz. Generally, the prompt installation of starship is less "sticky" than other approaches, but it has the advantage that it becomes very obvious if other unused prompts are computed in the background, which can have a performance impact.
